### PR TITLE
gis:store-dataset geojson support

### PR DESCRIPTION
**Pull request by JamesHovet**
Adds support for exporting geojson files in addition to just importing them

### CODE REVIEW

File: examples/geojson/geojson_export_test.nlogo [hunk 1]
```diff
@@ -0,0 +1,739 @@
...
+
+
+end
+
+to test-Point
+	clear-all
+	set dataset gis:load-dataset "Point.geojson"
+	gis:set-world-envelope (list -180 180 -90 90)
+	gis:set-drawing-color red
+	gis:draw dataset 1
```
Comment (LaCuneta) [inline examples/geojson/geojson_export_test.nlogo:39]: I know it's just a test file, but we do try to use 2 spaces in our NetLogo models for indentation (no tabs).  I see this in a few of the other procedures in this file, too.

File: src/org/myworldgis/io/geojson/GeoJsonReader.java [hunk 2]
```diff
@@ -23,10 +23,8 @@
 import org.json.simple.parser.JSONParser;
 import org.json.simple.parser.ParseException;
 
-public class GeoJsonReader {
+public class GeoJsonReader implements GeoJsonConstants {
```
Comment (LaCuneta) [inline src/org/myworldgis/io/geojson/GeoJsonReader.java:26]: That's an interest way to bring these constants in, I don't think I've seen that before.  It works for me.

**Review by LaCuneta — APPROVED** [review]
This mostly looks good to me, nice work.  I tested out with my national parks data and the export worked well.  I left one comment in the test nlogo file if you want to address that, otherwise this is good to merge.

File: src/org/myworldgis/io/geojson/GeoJsonReader.java (hunk 2 above)
Comment (JamesHovet) [reply src/org/myworldgis/io/geojson/GeoJsonReader.java:26]: I found it done that way in the shapefile importer code and thought I would keep it consistent.

**Pull request merged**